src/components/hh5p/player/index.tsx

```diff
@@ -75,14 +84,16 @@ export const Player: FunctionComponent<PlayerProps> = ({ id, onXAPI, styles = []
           {[...settings.core.scripts, ...settings.loadedJs].map((script) => (
             <script key={script} src={script}></script>
           ))}
-          {[...settings.core.styles, ...settings.loadedCss, ...styles].map((style) => (
-            <link
-              type="text/css"
-              rel="stylesheet"
-              key={style}
-              href={style}
-            ></link>
-          ))}
+          {[...settings.core.styles, ...settings.loadedCss, ...styles].map(
```

Comment:
now you have doubled this 

the url to css is in both `settings.core.styles` and `styles`